### PR TITLE
Somehow this failed on me

### PR DESCRIPTION
**Pull request by webpro**
Added the extra check and all is fine.

### CODE REVIEW

**Comment by yoniholmes** [comment]
Hi Lars, thanks for your pull request. I wanted to ask under which circumstances was this failing for you? As far as I understand, 'grunt.task.current.data' is always present when using grunt-text-replace, which is why it feels to be an unnecessary check. If you were able to get this to fail, I'm interested to know how to reproduce it.

Many thanks, 

Jonathan

**Comment by webpro** [comment]
A simple Grunt config excerpt:

```
replace: {
    testRunner: {
        src: ['test/index.html'],
        overwrite: true,
        replacements: [
            {
                from: 'FROM',
                to: 'TO'
            }
        ]
    }
}
```

When running this, in `grunt-text-replace.js` the `grunt.task.current.data` is `undefined` in `processGruntTemplate()`. However, it doesn't even reach the last line of that method:

```
return isProcessTemplateTrue ? grunt.template.process(string) : string;
```

That gives me:

```
Warning: Error while processing "test/index.html" file. Use --force to continue.
```

I guess the plugin uses `grunt.copy()`. But it actually did copy the file (despite the warning). Just tried to provide a `dest` instead of `overwrite:true`, but seems to result in the same error.

So, in short, it's not obvious to me what the actual issue is.

**Comment by yoniholmes** [comment]
Not sure the problem - I wasn't able to replicate that. Have merged your pull request in any case. Cheers, Jonathan

**Comment by webpro** [comment]
Thanks @yoniholmes! Appreciated.
